traffic_ops/traffic_ops_golang/systeminfo/system_info.go

```diff
@@ -87,15 +88,18 @@ WHERE p.config_file='global'`
 
 	info := make(map[string]string)
 	for rows.Next() {
-		p := tc.Parameter{}
+		p := tc.ParameterNullable{}
 		if err = rows.StructScan(&p); err != nil {
 			return nil, fmt.Errorf("getting system_info: %v", err)
 		}
-		if p.Secure && privLevel < auth.PrivLevelAdmin {
+		isSecure := *p.Secure
+		name := p.Name
+		value := p.Value
+		if isSecure && privLevel < auth.PrivLevelAdmin {
 			// Secure params only visible to admin
 			continue
 		}
-		info[p.Name] = p.Value
+		info[*name] = *value
```

Comment:
This will panic if name or value are null, but they haven't been checked anywhere.

Reply:
Noted: will make changes accordingly